proto-lens-tests/tests/proto3_test.hs

```diff
@@ -98,16 +98,20 @@ main = testMain
       , testCase "string" $ (def :: Strings) @=? (def & string .~ "")
       , testCase "enum" $ (def :: Foo) @=? (def & enum .~ Foo'Enum1)
       ]
-  -- Enums are all pattern aliases
+  -- Enums are sum types, except for aliases
   , testGroup "enum"
       [ testCase "aliases are exported" $ Foo'Enum2 @=? Foo'Enum2a
+      , serializeTo "serializeTo enum"
```

Comment:
Can you add a copy of this test which uses an unrecognized value?

Also: replace l.107 with more explicit generation:
```
tagged 6 $ VarInt 3
```
(where `6` is the proto tag of that field, and `3` is the value corresponding to `Enum2`)

Reply:
Ping on the first line of the above comment (unless I'm misreading the diff).

More specifically, please add this test which checks that unrecognized values round-trip through serialization as expected.
```
serializeTo "serializeTo enum unknown"
    (def & enum .~ toEnum 5 :: Foo)
    "enum: 5"
    $ tagged 6 $ VarInt 5
```

Reply:
whoops, sorry about that. Fixed.